### DISCUSSION TITLE
linux软件下载与帮助文档说明不符

### DISCUSSION THREAD

**Discussion by Prome-Theux** [natfrp/wiki#188] [使用问题] [answered]
我按照说明文档给出的命令下载了inux系统的软件（x86），运行命令后却发现有两个文件（frpc和frpc_linux_386）且只有后者可用。

  途中我遇到的问题：下载的时候总会只下载frpc文件，md5不对，且无法运行。后来多次运行安装命令后，才成功下载frpc_linux_386文件。这点说明文档并没有提出来，实际下载过程中给我带来了一些困扰。所以，可以完善一下说明文档吗？虽然这只是一个小问题。

**Comment by Archeb** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/11910831/198862948-0861fcc2-e30b-44da-9e2d-2818e988e8f4.png)

我这边按照[文档](https://doc.natfrp.com/#/frpc/usage)操作是没问题的，能否提供一下你的操作记录？

**Reply by Prome-Theux** [reply]
抱歉了，是我的问题。把-O打成小写的-o了
